prosper/common/prosper_logging.py

```diff
@@ -418,6 +402,7 @@ class DiscordWebhook(object):
 
     """
     __base_url = 'https://discordapp.com/api/webhooks/'
+    __webhook_url_format = __base_url + r"(\d+)/(\w+)"
```

Comment:
```python
__webhook_url_format = __base_url + r"(\d+)/(\w.+)"
```
not a fan of regular expressions (not expressive, not efficient), but this is a good application.  But `\w` only catches "words" and Discord's webhook format looks more like `[string]-[string]-[string]`.  This was causing issues when building url's.